libs/components/ag-grid/src/lib/modules/shared/sky-ag-grid-resources.module.ts

```diff
@@ -32,17 +32,23 @@ const RESOURCES: { [locale: string]: SkyLibResources } = {
       message: 'Editable currency {0} for row {1}',
     },
     sky_ag_grid_cell_renderer_number_validator_message: {
-      message: 'Please enter a valid number',
+      message: 'Enter a valid number.',
     },
     sky_ag_grid_cell_renderer_currency_validator_message: {
-      message: 'Please enter a valid currency',
+      message: 'Enter a valid currency.',
```

Comment:
Are these changes intentional or for testing?

Reply:
All of the `*-resources.module.ts` file changes are a result of the script fixing/updating their contents. So in this case, we had updated the resource JSON, but never ran `ng g @skyux/i18n:lib-resources-module` on `ag-grid` before releasing.

Reply:
In short, this change is intentional.